dom0/sd-sys-vms.sls

```diff
@@ -5,29 +5,42 @@
 # an up-to-date version of Fedora, in order to receive security updates.
 
 include:
-  # Import the upstream Qubes-maintained default-dispvm to ensure fedora-31-dvm
-  # is created.
+  # Import the upstream Qubes-maintained default-dispvm to ensure Fedora-based
+  # DispVM is created
   - qvm.default-dispvm
 
-{% set sd_supported_fedora_version = 'fedora-31' %}
+{% set sd_supported_fedora_version = 'fedora-32' %}
 
 # Install latest templates required for SDW VMs.
 dom0-install-fedora-template:
   pkg.installed:
     - pkgs:
       - qubes-template-{{ sd_supported_fedora_version }}
 
+hotfix-fedora-template-salt-packages:
+  cmd.wait:
+    - name: >
+        qvm-run {{ sd_supported_fedora_version }}
+        "sudo dnf update --refresh --enablerepo=qubes-vm-r4.0-current-testing -y qubes-mgmt-salt-vm-connector"
+        && qvm-shutdown --wait {{ sd_supported_fedora_version }}
+    - require:
+      - pkg: dom0-install-fedora-template
+    - watch:
+      - pkg: dom0-install-fedora-template
```

Comment:
The use of `wait` here means that the hotfix task will only run if the fedora-32 template has just been installed. For workstations that already have the fedora-32 template present, because someone installed it manually, outside the scope of Workstation provisioning logic. 

That's likely acceptable for our pilot participants, but in the future, for a larger install base, we'll likely want to script a solution: 

1. Check for tag "sd-workstation-salt-fix-2020-11" on fedora-32 template
2. If not present, run the hotfix task
3. Add tag if it was missing

I don't advocate for the script-based solution in this PR, @zenmonkeykstop, but mentioning it in case you think it's worthwhile based on pilot comms.